.github/actions/build-and-test/action.yml

```
@@ -67,6 +65,15 @@ runs:
       shell: bash
       run: ./gradlew docker
 
+    - name: Check Changed Images
+      id: changed-images
+      shell: bash
+      run: |
+        echo "Changed Projects: $(ls changed)"
+        for file in $(ls changed); do projects=(${projects[@]} "$file"); done; \
+        projects_array=$(jq --compact-output --null-input '$ARGS.positional' --args -- ${projects[@]}) \
+        && echo "projects=${projects_array}" >> "$GITHUB_OUTPUT"
```

Could this be simplified to:
```suggestion
      run: echo "projects=$(ls changed | jq --raw-input . | jq --slurp --compact-output .)" >> $GITHUB_OUTPUT
```